Sources/Apodini/Semantic Model Builder/Model/EndpointParameter.swift

```diff
@@ -153,18 +153,25 @@ public struct EndpointParameter<Type: Codable>: _AnyEndpointParameter, Identifia
         self.options = options
         self.necessity = necessity
         self.defaultValue = defaultValue
-
-        // If somebody wants to make this more fancy, one could add options into the @Parameter initializer
-        var description = "@Parameter var \(name): \(Type.self)"
+        
+        let httpOption = options.option(for: PropertyOptionKey.http)
+        var description = "@Parameter"
+        
+        if let httpOption = httpOption {
+            description += "(.http(.\(httpOption))"
```

Comment:
Yep exactly. I took a quick look into that too but then decided to leave it be.

Reply:
Of course i could just go by the available `PropertyOptionKey`s, so `.http`, `.mutability` etc. Not very future-proof, but yeah. Would that be acceptable to you?

Reply:
I'm honestly not sure what's best. Also don't have a strong opinion on that.
Maybe just concatenate the string version of all the option values, hoping that the values themselves are self explanatory to indicate the property key?

Not sure if it makes sense to blindly add options to the description at all? Maybe for a debugDescription? But the `description` is currently used in OpenAPI as a default (currently not overwritteable). Are the stringified options relevant at all?
I'm really not sure, just wanted to start the discussion, as you seemingly made some thoughts around that topic. But I'm also fine to leave it as you realized it.

Reply:
I'm not really a friend of just concatenate the string version of all the option values, seems just off and not really helpful to me, but if you think that's a good idea, I can definitely do that.

Another thought: Add a function (more swifty way: computed property) to `PropertyOptionKey` that returns the runtime type of `Option: PropertyOption` via `String(describing: ...)`. Would that be acceptable to you? (However I somehow guess we don't want to implement "the easy way out" :D)

And yep, probably makes more sense to do that in a debugDescription.

Reply:
I would have done something like that:
```swift
public class AnyPropertyOptionKey {
    var propertyOptionType: String {
        fatalError("not implemented")
    }
}

public class PropertyOptionKey<PropertyNameSpace, Option: PropertyOption>: AnyPropertyOptionKey {
    override var propertyOptionType: String {
        String(describing: Option.self)
    }
}
```

In case of `.http`, this results in the output "HTTPParameterMode", which is basically what we wanted, right?

Reply:
After making `PropertyOptionSet` iterable, we can do smt. like that: 
```swift
self.options.forEach({ key, value in
    print("\(key.propertyOptionType) = \(value)")
})
```

Resulting in:
"HTTPParameterMode = body
HTTPParameterMode = body"

for 

```swift
@Parameter(.http(.body))
var name: String? = "World"

@Parameter(.http(.body))
var newMove: CreateMoveMediator
```